docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+   "id": "946726ad",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Bringing Your Own Data (BYOD)?\n",
+    "\n",
+    "You can easily replace the above with your own multiannotator dataset, and continue with the rest of the tutorial.\n",
+    " \n",
+    "`multiannotator_labels` should be a numpy array or pandas DataFrame with each column representing an annotator and each row representing an example. Your classes (and entries of `multiannotator_labels`) should be represented as integer indices 0, 1, ..., num_classes - 1, where examples that are not annotated by a particular annotator are represented using `np.nan`.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51335def",
+   "metadata": {},
+   "source": [
+    "## 3. Get majority vote label and compute out-of-sample predicted probabilites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1857cc7",
+   "metadata": {},
+   "source": [
+    "Before training our machine learning model, we must first obtain the consensus labels from the annotators that labeled the data. The simplest way to obtain an initial set of consensus labels is to select it using majority vote."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d009f347",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "majority_vote_label = get_majority_vote_label(multiannotator_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7287b733",
+   "metadata": {},
+   "source": [
+    "Next, we will train our model on the consensus labels obtained using majority vote to compute out-of-sample predicted probabilities. Here, we use a simple logistic regression model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cbd1e415",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.model_selection import cross_val_predict\n",
+    "\n",
+    "model = LogisticRegression()\n",
+    "\n",
+    "num_crossval_folds = 5  \n",
+    "cv_pred_probs = cross_val_predict(\n",
+    "    estimator=model, X=X, y=majority_vote_label, cv=num_crossval_folds, method=\"predict_proba\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4eab5188",
+   "metadata": {},
+   "source": [
+    "## 4. Use cleanlab to get better consensus labels and other statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d392ce5",
+   "metadata": {},
+   "source": [
+    "Using the annotators' labels and the out-of-sample predicted probabilites from the model, cleanlab can help us obtain improved consensus labels for our data.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ca92617",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multiannotator_dict = get_label_quality_multiannotator(multiannotator_labels, cv_pred_probs)"
```

Comment:
Let's think carefully about the name of this variable `multiannotator_dict` since users tend to just copy the tutorials' code, so their own code will probably have this variable in it too (and thus should have clearest name possible).

Reply:
Alternative names could include:
`multiannotator_results` ,`multiannotator_analysis`, but I cant necessarily think of alternate names that are clearly superior to `multiannotator_dict`

Reply:
I like `multiannotator_results`!